packages/perseus/src/widgets/__tests__/radio_test.jsx

```diff
@@ -237,11 +241,23 @@ describe("single-choice question", () => {
             ).toHaveFocus();
         });
 
+        it("should be able to select an option by keyboard", () => {
+            // Arrange
+            renderQuestion(question, apiOptions);
+
+            // Act
+            userEvent.tab();
+            userEvent.keyboard("{space}");
+
+            // Assert
+            expect(screen.getAllByRole("radio")[0]).toBeChecked();
+        });
+
         it("should be able to navigate through 'None of the above' choice by keyboard", () => {
             // Arrange
             const q = clone(question);
             ((q.widgets["radio 1"]
-                .options: any): PerseusRadioWidgetOptions).choices[3].isNoneOfTheAbove = true;
+                .options: any): PerseusRadioWidgetOptions).choices[1].isNoneOfTheAbove = true;
```

Comment:
Changed this so that the test actually tabs through the None of the Above choice instead of to the choice based on the test title. Please let me know if this is not actually what the test is testing for though.

Reply:
I returned the value to its original state and updated the test title. Thanks! :)